src/styles/calculator.scss

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+$result-bg: rgb(133, 134, 147);
+$right-bg: rgb(245, 145, 63);
+$p-color: black;
+$result-color: white;
+$p-font: 'Poppins', sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: $p-font;
+}
+
+body,
+div,
+section,
+p {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  position: absolute;
+}
+
+.mainframe {
+  display: grid;
+  width: auto;
+  max-width: 500px;
+  grid-template-rows: repeat(6, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+
+  .btn {
+    height: 5rem;
+    width: 7rem;
+    border: 1px $result-bg solid;
+    box-sizing: border-box;
+  }
+
+  .orange {
+    background-color: $right-bg;
+  }
+
+  .result {
+    grid-area: 1 / 1 / 1 / 5;
+    background-color: $result-bg;
+    justify-content: flex-end;
+    box-sizing: border-box;
+    padding: 0 1rem;
+    color: $result-color;
+    font-size: 2rem;
+  }
+
+  .btn0 {
+    grid-area: 6 / 1 / 7 / 3;
+    border: 1px $result-bg solid;
+    box-sizing: border-box;
+  }
```

Comment:
- [ ] OPTIONAL: There are alignment issues in your buttons Panel. You may want to make changes in your spare time. 
![image](https://user-images.githubusercontent.com/64894927/128920301-4dd06cf0-fb57-46e6-8974-42dced5f0444.png)